pkg/config/user_config.go

```diff
@@ -513,10 +515,11 @@ func GetDefaultConfig() *UserConfig {
 				BulkMenu: "b",
 			},
 		},
-		OS:                   GetPlatformDefaultConfig(),
-		DisableStartupPopups: false,
-		CustomCommands:       []CustomCommand(nil),
-		Services:             map[string]string(nil),
-		NotARepository:       "prompt",
+		OS:                        GetPlatformDefaultConfig(),
+		DisableStartupPopups:      false,
+		CustomCommands:            []CustomCommand(nil),
+		Services:                  map[string]string(nil),
+		NotARepository:            "prompt",
+		DisplayConfirmationWindow: true,
```

Comment:
I think this could be named better given that we'll be displaying the window regardless (i.e. showing the output of the command as the command runs). How about 'PromptToReturnFromSubprocess'?